### ISSUE TITLE
wezterm-nightly fails to create a visible window on macOS 11.6.8 (Big Sur)

### ISSUE THREAD

**Issue by smferris** [wez/wezterm#2440]
### What Operating System(s) are you seeing this problem on?

macOS

### WezTerm version

20220820-223419-3917fc21

### Did you try the latest nightly build to see if the issue is better (or worse!) than your current version?

Yes, and I updated the version box above to show the version of the nightly that I tried

### Describe the bug

wezterm starts, WezTerm appears in the menu bar, my wezterm.lua is processed, but a window is never created.  The last output is:

2022-08-21 07:01:16.929 wezterm-gui[64043:12967899] -[NSScreen maximumFramesPerSecond]: unrecognized selector sent to instance 0x7f8b6af23a50

From the output, I'm guessing maximumFramesPerSecond doesn't exist in Big Sur, and wezterm isn't coping with that.

### To Reproduce

Run the wezterm-nightly homebrew cask on Big Sur 11.6.8.

### Configuration

I get the same result if I rename wezterm.lua out of the way, so the config doesn't matter.

### Expected Behavior

WezTerm creates a visible window.

**Comment by wez** [comment]
This should be fixed now in `main`.

It typically takes about an hour before fixes are available as nightly builds for all platforms.  Linux builds are the fastest to build and are often available within about 20 minutes.  Windows and macOS builds take a bit longer.

Please take a few moments to try out the fix and let me know how that works out. You can find the nightly downloads for your system in the [wezterm installation docs](https://wezfurlong.org/wezterm/installation.html).

If you prefer to use packages provided by your distribution or package manager of choice and don't want to replace that with a nightly download, keep in mind that you can download portable packages (eg: a `.dmg` file on macOS, a `.zip` file on Windows and an `.AppImage` file on Linux) that can be run without permanently installing or replacing an existing package, and can then simply be deleted once you no longer need them. 

If you are eager and can [build from source](https://wezfurlong.org/wezterm/install/source.html) then you may be able to try this out more quickly.

**Comment by smferris** [comment]
Verified fixed in 20220821-083319-f792a20f. Thanks for the quick fix.

**Comment by KyleKing** [comment]
I'm on macOS Mojave (10.14.6) and have the same type of issue where the WezTerm GUI isn't visible and a `NSScreen` error is reported:

```sh
> wezterm
2022-08-23 14:19:54.766 wezterm-gui[30185:1561789] -[NSScreen localizedName]: unrecognized selector sent to instance 0x7fdecd635c20

> wezterm --version
wezterm 20220821-195934-0702a716
```

I tried both stable (`20220807-113146-c2fee766`) and nightly (`wezterm 20220821-195934-0702a716` which is slightly newer than what was reported to work on macOS Big Sur)
